src/AppBundle/Controller/WebhookController.php

```diff
@@ -21,6 +21,17 @@ public function githubAction(Request $request)
             throw new \Exception('Invalid JSON body!');
         }
 
+        if (!isset($data['repository']['full_name'])) {
+            throw new \Exception('Repository name not defined!');
+        }
+
+        list($vendor, $name) = explode('/', $data['repository']['full_name']);
+        $repository = $this->get('repository_provider')->getRepository($vendor, $name);
+        if (null === $repository) {
+            throw $this->createNotFoundException(sprintf('Repository %s/%s not found.', $vendor, $name));
```

Comment:
"not found" sounds like it doesn't exists (on GitHub). What about saying "Repository "%s/%s" is not enabled." or "Repository "%s/%s" is not supported"?